package.json

```diff
@@ -34,7 +34,7 @@
     "test:coverage": "npm run test:only -- --collectCoverageFrom=\"src/**/*.js\" --coverage",
     "pretest": "npm run lint",
     "test": "npm run test:coverage",
-    "prepare": "husky install && npm run build",
+    "prepare": "husky && npm run build",
```

Comment:
Is this needed? Why?

Reply:
I guess the change from the new version runs install when husky is invoked?

Reply:
This is mentioed in the [migration guide](https://github.com/typicode/husky/releases/tag/v9.0.1), `install` is no longer required & deprecated.

<img width="623" alt="Screenshot 2024-07-28 at 9 16 06 PM" src="https://github.com/user-attachments/assets/c588f549-9c86-4588-9043-a5a6096daa8c">